### PR TITLE
doc: add security advisory section in ACRN introduction website

### PR DESCRIPTION
**Pull request by yonghuah**
Add survivability plan for ACRN Open source project.

Tracked-On: #2945
Signed-off-by: Yonghua Huang <yonghua.huang@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by yonghuah** [comment]
@dbkinder  Can help to review this patch?  it is for survivability  plan for ACRN open source project.

**Comment by wenlingz** [comment]
@dbkinder  please help review and merge this PR,thanks

File: README.rst [hunk 1]
```diff
@@ -64,6 +64,22 @@ ACRN support systems:
   the Github issues system: https://github.com/projectacrn/acrn-hypervisor/issues.
   You can browse through the reported issues and submit issues of your own.
 
+* **Reporting a Potential Security Vulnerability**: If you have discovered potential
+  security vulnerability in ACRN, please send an e-mail to secure@intel.com. For issues
+  related to Intel Products, please visit https://security-center.intel.com.
+
+  It is important to include the following details:
+  - The projects and versions affected
```
Comment (dbkinder) [inline README.rst]: add a blank line before the first element of a list

Comment (yonghuah) [reply README.rst]: @dbkinder have updated, please review ,thanks.
-Yonghua

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**